migrations/0003_a11y_structure_updates.py

```diff
@@ -0,0 +1,32 @@
+from django.db import migrations
+from customstyling.utils import find_and_handle_custom_stylesheets
+
+
+def forward_migration(apps, schema_editor):
+    changes_to_make = [
+        ('olh', '#content aside', '#content .side-info'),
+        ('olh', '#content .side-info h3', '#content .side-info h2'),
+    ]
+    find_and_handle_custom_stylesheets(apps, schema_editor, changes_to_make, reverse=False)
+
+
+def reverse_migration(apps, schema_editor):
+    changes_to_make = [
+        ('olh', '#content aside', '#content .side-info'),
+        ('olh', '#content aside h3', '#content aside h2'),
```

Comment:
be mindful of having some entries as substrings of one another. In this case, we probably want to run the second string ahead of the first one, as the first string is a substring and would not allow the other one to run at all